src/predictors/ReLU.jl

```diff
@@ -244,7 +244,7 @@ function add_predictor(
 end
 
 """
-    ReLUQuadratic() <: AbstractPredictor
+    ReLUQuadratic(; relaxation_parameter = nothing) <: AbstractPredictor
```

Comment:
I'm okay with the name, but other options could be: `relaxation_tolerance`, `tolerance`, `atol`, `complementarity_mu`, `mu`, `complementarity_tolerance`... not sure if any of those are better

Reply:
I like `relaxation_*` as it communicates the qualitative change that happens when this parameter is set. (Although I may kick myself for this when I inevitably forget what this is called in six months.) I could go with `tolerance` or `parameter`.